src/wrappers/lambda.js

```diff
@@ -190,11 +190,8 @@ function baseLambdaWrapper(
             if (statusCode) {
                 eventInterface.addToMetadata(runner, { status_code: statusCode });
             }
-
-            const table = utils.getObjectFromRunner(runner);
             const config = getConfig();
-
-            if (error === null && !config.metadataOnly && !config.ignoredDBTables.includes(table)) {
+            if (error === null && !config.metadataOnly && !config.skipReturnValue) {
```

Comment:
maybe returnValueEnabled will be better, to save us the headache of the double negation?